src/meta-srv/src/bootstrap.rs

```diff
@@ -461,6 +461,7 @@ fn build_connection_options(tls_config: Option<&TlsOption>) -> Result<Option<Con
     if matches!(tls_config.mode, TlsMode::Disable) {
         return Ok(None);
     }
+    info!("Creating etcd client with TLS config: {:?}", tls_config);
```

Comment:
Is it safe to print this?

Reply:
[4b793b0](https://github.com/GreptimeTeam/greptimedb/pull/6941/commits/4b793b0df4cc8ca8010f5ad46f1e489df64e5e8e)